contrib/crates.sh

```diff
@@ -5,4 +5,4 @@
 # shellcheck disable=SC2034
 
 # Crates in this workspace to test (note "fuzz" is only built not tested).
-CRATES=("addresses" "base58" "bitcoin" "fuzz" "hashes" "internals" "io" "units")
+CRATES=("addresses" "base58" "bitcoin" "primitives" "fuzz" "hashes" "internals" "io" "units")
```

Comment:
The real process mega fail is that we're no longer using automated way to get the list of the crates. I know you said this PR is not perfect but I'd really, really like to see it fixed right here rather than kicking it down the road. You should be able to just copy the command from old revisions. And regarding fuzz order, I suggest to just patch-out the crate from the list and then explicitly add it to the end (why isn't it at the end already?)